sktime/utils/mlflow_sktime.py

```diff
@@ -0,0 +1,534 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+"""The ``mlflow_sktime`` module provides an MLflow API for ``sktime`` models.
+
+This module exports ``sktime`` models in the following formats:
+
+sktime (native) format
+    This is the main flavor that can be loaded back into sktime, which relies on pickle
+    internally to serialize a model.
+mlflow.pyfunc
+    Produced for use by generic pyfunc-based deployment tools and batch inference.
+
+    The ``pyfunc`` flavor of the model supports sktime predict methods ``predict``,
+    ``predict_interval``, ``predict_quantiles`` and ``predict_var`` (``predict_proba``
+    and ``predict_residuals`` are currently not supported).
+"""
+
+__author__ = ["benjaminbluhm"]
+__all__ = [
+    "get_default_pip_requirements",
+    "get_default_conda_env",
+    "save_model",
+    "log_model",
+    "load_model",
+]
+
+import logging
+import os
+import pickle
```

Comment:
is it possible to use `cloudpickle` instead `pickle`? Or even parametrize this so user could chose?

Reply:
Let me check. 

What would be the advantage of using `cloudpickle` or `parametrization` this?